roadmap/6_infrastructure/in-progress/4_Observability.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Observability
+links:
+  Forum Post:
+  Proposal:
+eta:
+is_community: false
+---
+
+We plan to specify and build a second-generation observability system that addresses shortcomings of the current system and prepares us for new challenges ahead.
```

Comment:
This is _very_ generic. Is there any way you can add at least a few more concrete things? Like what's the problem with the current one?

Reply:
Hi Severin, this was the first  pull request to test out the working of the roadmap functionality. I will start gradually adding more content and details for the Infrastructure Roadmap. Observability is the first official feature that is on feature planning but was not included in the roadmap planning yet.